package/gluon-mesh-vpn-core/luasrc/lib/gluon/upgrade/500-mesh-vpn

```diff
@@ -12,6 +12,7 @@ uci:section('network', 'interface', 'mesh_vpn', {
 	ifname = vpn_core.get_interface(),
 	proto = 'gluon_mesh',
 	fixed_mtu = true,
+	hop_penalty = uci:get('gluon', 'batadv_hop_penalty', 'mesh_vpn') or 0,
```

Comment:
Why is the `or 0` here necessary? Can we not leave the field unset and handle the missing value in `gluon_mesh.sh`? Ideally, there should be only one piece of code that sets the default to 0, and all other places just forward the (set or unset) value the the next layer.

Reply:
I need to double check this, but will put my testing efforts into triband at first.